finally 절은 try문 수행 도중 예외 발생 여부에 상관없이 수행된다.